sdk/eventhub/eventhubs-checkpointstore-table/package.json

```
@@ -21,7 +21,7 @@
   "module": "dist-esm/src/index.js",
   "types": "./typings/eventhubs-checkpointstore-table.d.ts",
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=8.0.0"
```

Oh, this actually should be 
```suggestion
    "node": ">=12.0.0"
```